samples/snippets/fsharp/tour.fs

```diff
@@ -71,7 +71,7 @@ module IntegersAndNumbers =
 /// Values in F# are immutable by default.  They cannot be changed
 /// in the course of a program's execution unless explicitly marked as mutable.
 ///
-/// To learn more, see: https://docs.microsoft.com/dotnet/articles/fsharp/language-reference/values/index#why-immutable
+/// To learn more, see: https://docs.microsoft.com/dotnet/fsharp/language-reference/values/index#why-immutable
```

Comment:
I thought we were leaving *index* off when going to *index.md* ???

*https://docs.microsoft.com/dotnet/fsharp/language-reference/values/#why-immutable*